bundles/admin/admin-layereditor/components/Collapse.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Collapse from 'antd/lib/collapse';
```

Comment:
We could name Collapse as AntCollapse and then export our CollapseComponent as Collapse instead.